src/D2L.Bmx/Program.cs

```diff
@@ -27,10 +27,12 @@
 		result.ErrorMessage = "Invalid duration";
 	}
 } );
-var nonInteractiveOption = new Option<bool>(
+var nonInteractiveFlag = new Option<bool>(
 	name: "--non-interactive",
-	getDefaultValue: () => false,
```

Comment:
Wanted to get rid of this and do the same for the new one. My thought was how can I know whether or not the user did not include `--allow-project-configs` option or if they did `--allow-project-configs false`. And depending on that how do I know whether I should prompt them in the configure command for this value if the value was false

Reply:
So now it's just if the flag is present that implies the value is true

Reply:
I thought to rename it too but was 50/50 on whether we should keep to the format we have for everything else or include `Flag` to make it a bit more obvious

Reply:
without this, what happens when user doesn't supply the flag? what does `GetValueForOption` return?

Reply:
I'd like to keep calling of them "option" for consistency. I don't think "flag" makes things more obvious here.

Reply:
It's false if they don't supply the option

Reply:
So `getDefaultValue: () => false` isn't doing anything anyway? Yeah good to get rid of it here.